lib/fishjam/rpc_client.ex

```diff
@@ -51,4 +65,9 @@ defmodule Fishjam.RPCClient do
   end
 
   defp nodes, do: [Node.self() | Node.list()]
+
+  defp emit_rpc_duration_event(event_name, start_time) do
+    duration = System.monotonic_time(:millisecond) - start_time
```

Comment:
What do you think about adding a function `get_time`, so we will guarantee the time units to be the same in this module?